editor/configs/unitColumns.tsx

```diff
@@ -39,7 +39,12 @@ export const ALL_UNIT_COLUMNS = {
     editable: true,
     inputType: "number",
     inputOptions: {
-      validate: commonValidations.isNumber,
+      validate: commonValidations.isPositiveWholeNumber,
+      min: { value: 1900, message: "Year must be 1900 or later" },
```

Comment:
currently you can get around this by typing something like 2a and it will just save as 2